krizten/Memoire-WebClient
# #165541556 Develop Not Found Page UI

### What does this PR do?
- Build out the page for non-matching routes.
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the login page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ch/165541556/not-found-page
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/not-found``` or any non-matching route to see the not-found (404) page.
#### What are the relevant pivotal tracker stories?
#165541556
#### Screenshots (if appropriate)
![page404-desktop](https://user-images.githubusercontent.com/29978455/57364032-36bfd080-717a-11e9-85db-31d41fbfa4da.png)
![page404-mobile](https://user-images.githubusercontent.com/29978455/57364046-3d4e4800-717a-11e9-89d7-f61221559434.png)
